packages/auth/src/providers/cognito/utils/signInStore.ts

```diff
@@ -111,32 +108,32 @@ const getDefaultState = (): SignInState => ({
 
 // Hydrate signInStore from Synced Session Storage
 const initializeState = (): SignInState => {
-	if (!syncSessionStorage) {
-		return getDefaultState();
-	}
-
 	const expiry = syncSessionStorage.getItem(signInStateKeys.expiry);
-	if (isExpired(expiry)) {
-		logger.warn('Sign-in session expired');
-		clearPersistedSignInState();
+	if (expiry) {
+		if (isExpired(expiry)) {
```

Comment:
you can collapse the 2 if statements into one

```suggestion
if (expiry&&isExpired(expiry)) {
```